packages/component/src/Activity/Timestamp.js

```diff
@@ -3,11 +3,16 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 import connectToWebChat from '../connectToWebChat';
+import AbsoluteTime from '../Utils/AbsoluteTime';
 import RelativeTime from '../Utils/RelativeTime';
 
 const Timestamp = ({ activity: { timestamp }, className, styleSet }) => (
   <span className={classNames(styleSet.timestamp + '', (className || '') + '')}>
-    <RelativeTime value={timestamp} />
+    {styleSet.options.timestampConfig === 'relative' ? (
```

Comment:
Do we need parenthesis here? Or added by ESLint/Prettier?

```suggestion
    {styleSet.options.timestampConfig === 'relative' ?
```

Reply:
ESLint/Prettier added the parenthesis, and I only know this because I would have been to impatient to add them myself...